src/Recoil_index.js

```diff
@@ -34,6 +34,10 @@ export type {
   SelectorFamilyOptions,
 } from './recoil_values/Recoil_selectorFamily';
 
+const {unstable_batchedUpdates} = require('./util/ReactBatchedUpdates');
+const {setBatch} = require('./util/Recoil_batch');
+setBatch(unstable_batchedUpdates);
```

Comment:
Code should be placed after the `require` blocks.  There should be a prettier rule for this.  Since this is adding actual logic to `Recoil_index.js`, please add comments to explain.  It looks like this is meant to be for production while `setupReactBatching.js` is for testing?  Could this be moved to `Recoil_batch.js`?

Reply:
I've moved this to `Recoil_batching.js`. 👌